### PR TITLE
Update kubectl cheat sheet with command to identify Nodes in Ready state

### PR DESCRIPTION
**Pull request by prasannagithub**
Added - Check which nodes are ready with custom-columns

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle Update kubectl cheat sheet with command to identify Nodes in Ready state

File: content/en/docs/reference/kubectl/cheatsheet.md
Comment (Ritikaa96) [inline content/en/docs/reference/kubectl/cheatsheet.md:219]:
```suggestion
kubectl get node -o custom-columns='NODE_NAME:.metadata.name,STATUS:.status.conditions[?(@.type=="Ready")].status'
```

Comment (Ritikaa96) [reply content/en/docs/reference/kubectl/cheatsheet.md:219]: The current version is good but Wouldn't it be better to see `Node Age` alongside with status?

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/kubectl/cheatsheet.md
Comment (prasannagithub) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Thanks a lot for your suggestion, Is it ok to allow this change as it is and latter I will add your suggestion as well. thanks a lot again

**Comment by Ritikaa96** [comment]
This LGTM
/lgtm

**Comment by sftim** [comment]
Hi @prasannagithub 

Why should we make this change? You've not said.

**Comment by prasannagithub** [comment]
@sftim After going through the "kubectl Cheat Sheet", I thought that "custom-columns" should be there right after the "jsonpath", that is my whole objective as well as that the user can construct many more use cases based on the given command.

**Comment by sftim** [comment]
I suggest we first demonstrate implement a basic custom column example (to appear earlier). This is handy but quite complex! I'd like to build up to this with a hint that helps people who really just want a custom column and don't plan to grok the JSONPath part.

Maybe for Pods or Services, where custom views can be handy.

Would you be willing to change the PR accordingly @prasannagithub?

**Comment by Ritikaa96** [comment]
> I suggest we first demonstrate implement a basic custom column example (to appear earlier). This is handy but quite complex! I'd like to build up to this with a hint that helps people who really just want a custom column and don't plan to grok the JSONPath part.
> 
> Maybe for Pods or Services, where custom views can be handy.
> 
> Would you be willing to change the PR accordingly @prasannagithub?


Hi , there is already a section dedicated to Formatted outputs [here](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#formatting-output)

Examples using -o=custom-columns:

```go

# All images running in a cluster
kubectl get pods -A -o=custom-columns='DATA:spec.containers[*].image'

# All images running in namespace: default, grouped by Pod
kubectl get pods --namespace default --output=custom-columns="NAME:.metadata.name,IMAGE:.spec.containers[*].image"

 # All images excluding "registry.k8s.io/coredns:1.6.2"
kubectl get pods -A -o=custom-columns='DATA:spec.containers[?(@.image!="registry.k8s.io/coredns:1.6.2")].image'

# All fields under metadata regardless of name
kubectl get pods -A -o=custom-columns='DATA:metadata.*
```

**Comment by Ritikaa96** [comment]
This addition is in [Viewing and Finding Resources](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#viewing-and-finding-resources) section. 
it kind of continues the previous example where we 
```
# Check which nodes are ready
JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
 && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
```

Were you thinking in same way while adding this example @prasannagithub ?

**Comment by prasannagithub** [comment]
hi @Ritikaa96  Yes, you are correct, It give us a human readable output. 

Hi All, Please kindly approve this small change please. Thanks a lot

**Comment by Ritikaa96** [comment]
> Hi All, Please kindly approve this small change please. Thanks a lot
@prasannagithub asking for approval is not recommend as per community [guidelines](https://kubernetes.io/docs/contribute/)
Let's wait for the maintainers to review this PR and provide the approvals .

**Comment by prasannagithub** [comment]
@Ritikaa96 thanks a lot for all the help & guideline

**Comment by Ritikaa96** [comment]
> @Ritikaa96 thanks a lot for all the help & guideline

That's no issue. Please just keep the guidelines in mind.

**Pull request merged**
